src/openzaak/notifications/handler_objecten.py

```diff
@@ -0,0 +1,82 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2020 Dimpact
+from urllib.parse import urlparse
+
+from django_loose_fk.loaders import BaseLoader
+from vng_api_common.constants import CommonResourceAction
+
+from openzaak.components.zaken.models import Zaak, ZaakObject
+
+
+def handle(message: dict) -> None:
+    kanaal = message.get("kanaal")
+    if kanaal != "objects":
+        return
+
+    resource = message.get("resource")
+    actie = message.get("actie")
+    resource_url = message.get("resourceUrl")
+    kenmerken = message.get("kenmerken", {})
+    objecttype_omschrijving = kenmerken.get("objecttypeOmschrijving")
+    zaken_urls = message.get("zaken", [])
+
+    if resource != "object":
+        return
+
+    loader = BaseLoader()
+    resolved_zaken_ids = []
+
+    for url in zaken_urls:
+        # Resolve local zaak URLs
+        for url in zaken_urls:
+            parsed = urlparse(url)
+            if parsed.netloc == "openzaak.example.com":
```

Comment:
This is now hardcoded to compare to `openzaak.example.com`. You can use `BaseLoader.is_local_url` instead to check if the URL is local: https://github.com/maykinmedia/django-loose-fk/blob/c9acff87f0c9dae8cc2cd707bb043a85e74eda6b/django_loose_fk/loaders.py#L33